app/views/prior_authority/applications/index.html.erb

```diff
@@ -24,20 +24,20 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
 
-      <%= govuk_tabs(title: t(".page_title")) do |c| %>
+      <%= govuk_tabs(title: t(".page_title"), classes: ['reload-visible-turboframes'], html_attributes: { data: { selector: '.govuk-tabs__tab' }}) do |c| %>
         <% c.with_tab(label: t(".tabs.assessed")) do %>
-          <%= turbo_frame_tag 'assessed_applications', src: assessed_prior_authority_applications_path do %>
+          <%= turbo_frame_tag 'assessed_applications', src: assessed_prior_authority_applications_path, loading: :lazy do %>
             <%= render 'prior_authority/applications/frame_contents/assessed', model: @assessed_model, pagy: @assessed_pagy %>
           <% end %>
         <% end %>
         <% c.with_tab(label: t(".tabs.submitted")) do %>
-          <%= turbo_frame_tag 'submitted_applications', src: submitted_prior_authority_applications_path do %>
+          <%= turbo_frame_tag 'submitted_applications', src: submitted_prior_authority_applications_path, loading: :lazy do %>
             <%= render 'prior_authority/applications/frame_contents/submitted', model: @submitted_model, pagy: @submitted_pagy %>
           <% end %>
         <% end %>
         <% c.with_tab(label: t(".tabs.drafts")) do %>
-          <%= turbo_frame_tag 'draft_applications', src: draft_prior_authority_applications_path do %>
-            <%= render 'prior_authority/applications/frame_contents/draft', model: @draft_model, pagy: @draft_pagy %>
+          <%= turbo_frame_tag 'draft_applications', src: draft_prior_authority_applications_path, loading: :lazy %>
```

Comment:
Is this working for you? I'm running it locally and getting this:
<img width="1038" alt="Screenshot 2024-02-26 at 17 20 02" src="https://github.com/ministryofjustice/laa-submit-crime-forms/assets/113888736/ed642df0-0510-49d6-bad7-862194228ecc">

Reply:
Looks to me that you have a `do` in the wrong place.

Reply:
That is on me, I made a change at the end to make it work with JS disabled and didn't double check the view...

I had put the `do` on the wrong line which is why it rendered... this is now fixed.